src/components/DocumentationTopic.vue

```diff
@@ -300,6 +301,22 @@ export default {
     ),
     shouldShowLanguageSwitcher: ({ objcPath, swiftPath }) => objcPath && swiftPath,
     hideSummary: () => getSetting(['features', 'docs', 'summary', 'hide'], false),
+    enableHero: ({ symbolKind }) => (symbolKind ? (symbolKind === 'module') : true),
```

Comment:
I cant verify from a data perspective if this is correct. @mportiz08 wdyt?

Reply:
This is meant for top level technologies page that has a symbol kind (module). Maybe Marcus should confirm if this is the correct logic for displaying dark hero background(enable) for the 4 types of pages we are interested in.